pytd/client.py

```diff
@@ -161,6 +162,7 @@ def load_table_from_dataframe(self, dataframe, table, if_exists='error'):
             What happens when a target table already exists.
         """
         if self.writer is None:
+            self.managed_writer = True
```

Comment:
You can delete this line when the flag is initialized by `writer is None` in `__init__`

Reply:
ditto